### PR TITLE
added the app theme

### PR DESCRIPTION
**Pull request by PhucNgo88**
App theme is now added.

### CODE REVIEW

File: app/src/main/res/layout/fragment_start_scan.xml [hunk 1]
```diff
@@ -15,6 +14,8 @@
         android:layout_marginEnd="8dp"
         android:layout_marginStart="8dp"
         android:layout_marginTop="8dp"
+        android:background="@android:color/darker_gray"
```
Comment (Lionel1232) [inline app/src/main/res/layout/fragment_start_scan.xml]: Maybe this line can be removed, isn't the background being set by the theme?

**Comment by Lionel1232** [comment]
2018_09_11_23_27_09_window

This is what I mean about the buttons, maybe we should just leave it, what do you think @Casper-P

**Comment by Casper-P** [comment]
I have the same comments as @Lionel1232 in regards to altering the format of the button to look a little bit neater, while also changing the background colour to a darker shade as the white is very bright with the black and orange toolbars.

File: app/src/main/res/layout/fragment_start_scan.xml (hunk 1 above)
Comment (suborbitalbacon) [reply app/src/main/res/layout/fragment_start_scan.xml]: Removing this forced background colour will also solve the non-rounded button problem.

**Comment by PhucNgo88** [comment]
I've changed the background colour to white @suborbitalbacon . What do you suggest we do for the styling of the buttons?

**Pull request merged**